doc/release-notes-26076.md

```diff
@@ -0,0 +1,9 @@
+RPC
+---
+
+- The `deriveaddresses`, `getdescriptorinfo` and `listdescriptors` RPC's now
+  use `h` rather than apostrophe (`'`) to indicate hardened derivation. This
+  makes it easier to handle descriptor strings manually. E.g. an RPC call that
```

Comment:
The enclosing single quotes are shell escaping, not part of the actual RPC call (which has no issue at present).

Reply:
By "manually" I mostly mean using `bitcoin-cli` from the shell.